### PR TITLE
First pass at /about

### PR DESCRIPTION
**Pull request by joemasilotti**
Still need to tweak the copy here, but I think it's coming along nicely!

### CODE REVIEW

**Comment by joemasilotti** [comment]
This is ready to deploy code-wise, but I'd love a second/third pass on the copy.

**Comment by joemasilotti** [comment]
This PR also removes `lefthook`.

In theory, I love the idea of running `standardrb` on commit. In practice, it is too slow and gets in the way of my workflow. The alternative is to install a linter in your text editor or IDE. I'm using [ale](https://github.com/dense-analysis/ale) in my vim config.

CC @lsparlin

File: app/views/about/show.body.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Empowering the developer
+
+I've been an independent developer for the past two years. And leads are rarely consistent. Sometimes I can barely keep up with the work and other times I struggle to find my next gig.
+
```
Comment (vamsipavanmahesh) [inline app/views/about/show.body.md:4]: I know you are the owner of the project, but at this point, this is a community project right, suggest you remove the wording "I". Just my opinion

Comment (vamsipavanmahesh) [reply app/views/about/show.body.md:4]: actually, never mind, this might not be a true community project yet like rails.

**Pull request merged**